Release candidate 0.9.0

jschnasse opened the pull request:
Gemeinsamer, konsolidierter PullRequest für 
- https://github.com/edoweb/regal-api/pull/123
- https://github.com/edoweb/regal-api/pull/124
- https://github.com/edoweb/regal-api/pull/126
- https://github.com/edoweb/regal-api/pull/128

Löst:
https://jira.hbz-nrw.de/browse/EDOZWO-677 siehe Beschreibung im Jira ganze unten.
https://jira.hbz-nrw.de/browse/EDOZWO-676
https://jira.hbz-nrw.de/browse/EDOZWO-675
https://jira.hbz-nrw.de/browse/EDOZWO-646

Ist bereits auf test deployed!

Commit:
parametrize maximum number of crawls per night

Commit:
create new endpoint importVersion

Commit:
Neuer Endpunkt importVersion

Commit:
change default crawler-beans configuration

Commit:
change default crawler-beans configuration

Commit:
change default crawler-beans configuration

Commit:
activate scope logging
disable TransclusionDecideRule

Commit:
change date calculation for outstanding webcrawls

Commit:
replace exception mail by error message

Commit:
get application parameter in Globals

Commit:
use application parameter read from Globals

Commit:
Merge branch 'EDOZWO-646' of https://github.com/inkuss/regal-api into release_candidate_v2

Commit:
Merge branch 'EDOZWO-675' of https://github.com/inkuss/regal-api into release_candidate_v2

jschnasse committed:
add access information to parts
- similar to files

Commit:
Merge branch 'EDOZWO-677' of https://github.com/edoweb/regal-api into release_candidate_v2

Commit:
eliminate reference to play.Play in Create.java
eliminate changes which belong to EDOZWO-675

Commit:
Merge branch 'EDOZWO-676' of https://github.com/inkuss/regal-api into release_candidate_v2

inkuss commented:
Bisher können davon nur einzelne Teile sicher übernommen werden: #124, #128

inkuss commented:
Nach weiteren Analysen und Tests auf edoweb-test (siehe Kommentare an den einzelnen Unter-PullRequests) kann aus meiner Sicht nun auch der Bestandteil #126 sicher übernommen werden.
Bei #123 gibt es jedoch Probleme.
@aquast : den Bestandteil #128 bitte noch einmal inhaltlich testen bzw. testen lassen.

aquast commented:
Hi, ich kann der Diskussion gerade noch nicht ganz folgen. reviewe aber. Derzeit werfen Webseiten 500er Fehler, wenn die wayback aufgerufen wird. Ist das ein Fehler?

Commit:
take pid from Node.getPid() not from conf.getPid()

inkuss commented:
!!ACHTUNG !! Anstatt #123 muss der AKTUELLE STAND von inkuss/regal-api/EDOZWO-646 übernommen werden !!! Keine Ahnung wie das in git geht.
An dem Branch EDOZWO-646 musste ich heute Änderungen machen, sonst läuft die Webgatherer Sequenz  nicht mehr.

Commit:
Merge branch 'EDOZWO-646' of https://github.com/inkuss/regal-api into release_candidate_v2

jschnasse commented:
Neue Version wird auf test ausgerollt.

jschnasse committed:
add class file and part to distinguish types of object parts

jschnasse committed:
fix: waybacklink is generated on read time
- this seems to be wrong. In future it generated at
creation time
- for old versions with waybacklink == null a dynamic
creation is still in place.

jschnasse committed:
Distinguish object types in part view
- in order to provide special handling of webpage
versions

jschnasse committed:
Distinguish object types in part view
- in order to provide special handling of webpage
versions

jschnasse committed:
open extern links in new tab/window

jschnasse committed:
add webgatherer config in any case
- fix links if no config is available

jschnasse committed:
create waybacklink if version does not provide one
- fallback to startDate

jschnasse committed:
use /all html view as normal view
- provides display for all direct children

jschnasse committed:
add fallback if creator or publisher is not present

inkuss committed:
Update Helper.java

jschnasse committed:
Merge pull request #130 from edoweb/EDOZWO-677_webpages

Edozwo 677 webpages

jschnasse committed:
Merge branch 'release_candidate_v2' of https://github.com/edoweb/regal-api into EDOZWO-677_externLinks

Conflicts:
	app/actions/Read.java
	app/views/Helper.java
	app/views/tags/partsView.scala.html

jschnasse committed:
Merge pull request #131 from edoweb/EDOZWO-677_externLinks

jschnasse committed:
Merge pull request #132 from edoweb/FRL-143

add fallback if creator or publisher is not present

jschnasse commented:
dieser Stand ist nun auf test @inkuss

jschnasse committed:
Merge remote-tracking branch 'origin/master' into release_candidate_v2
